src/librustc/diagnostics.rs

```diff
@@ -675,12 +676,9 @@ E0269: r##"
 Functions must eventually return a value of their return type. For example, in
 the following function:
 
-```compile_fail
-fn foo(x: u8) -> u8 {
-    if x > 0 {
-        x // alternatively, `return x`
-    }
-    // nothing here
+```compile_fail,E0269
+fn abracada_FAIL() -> String {
+    "this won't work".to_string();
```

Comment:
Is this change intentional?
It seems the source code does not match the following explanation text anymore.